### DISCUSSION TITLE
I can get banned for having too many accounts authenticated using SentryFileHash?

### DISCUSSION THREAD

**Discussion by Cappi1998** [SteamRE/SteamKit#1003] [General]
i know if i have many accounts connected at the same time i can get banned like 500+ accounts for example. 

but my question is about Sentry FileHash, I am using it to connect without having to generate a new mobile vapor guard code.

EXAMPLE: If I authenticate 10,000 accounts ("Not keeping them online at the same time"), can I get a ban? Steam can ban me for having multiple accounts authenticated on the same machine, even if they are NOT online at the same time.

from what i understand steam tracks MachineGuid + serial hdd + mac am i right?

Sentry File Hash is responsible for this? The only description I found about it was "// machine authentication data."

**Comment by LennonSouza** [comment]
good question, I would like to know that too!

I had 300 banned community accounts, all on the same computer and the same ip.

**Comment by yaakov-h** [comment]
Valve can ban you for whatever they feel like, subject to their terms of service.

We can never guarantee that anything will avoid a ban and cannot condone circumventing bans or other security restrictions.

**Comment by Cappi1998** [comment]
I spent 3 days working on an update, and now I'm afraid to use it 😂😂
